### PR TITLE
vrs rocksdb snapshotting

### PR DESCRIPTION
**Pull request by theferrit32**

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -18,7 +18,11 @@
 
 
   :main-dev
-  {:main-opts ["-e" "(require 'genegraph.main)"
+  {:jvm-opts ["-Xmx256m"
+              "-XX:NativeMemoryTracking=detail"]
```
Comment (toneillbroad) [inline deps.edn:22]: Curious about why you choose this option.

**Review by toneillbroad — APPROVED** [review]

File: deps.edn (hunk 1 above)
Comment (theferrit32) [reply deps.edn:22]: I thought `NativeMemoryTracking` might give some visibility into native memory allocations being done by Rocks DB, but I don't think it does. I'll remove that option.

Comment (theferrit32) [reply deps.edn:22]: https://docs.oracle.com/en/java/javase/20/vm/native-memory-tracking.html

**Comment by theferrit32** [comment]
Closes #763

**Pull request merged**